### ISSUE TITLE
i18nTextCollectorTask error

### ISSUE THREAD

**Issue by brandoniffert** [kinglozzer/silverstripe-metatitle#12]
Currently in SilverStripe 4.3.0, when I run `i18nTextCollectorTask` to generate my locale text files, I get the following error:

```
ERROR [User Notice]: Missing localisation default for key Title
IN GET dev/tasks/i18nTextCollectorTask
```

It is referring to this line in `MetaTitleExtension.php`

```
$labels['MetaTitle'] = _t(SiteTree::class.'.METATITLE', 'Title');
```

(Same issue with `SiteTree::class.'.METATITLEHELP'` in `updateCMSFields`)

It seems like the way you are including the `SiteTree::class` string is tripping up SilverStripe (possibly because of the unescaped backslashes)

Replacing these `SiteTree::class` instances with `SilverStripe\\CMS\\Model\\SiteTree.METATITLE` and `SilverStripe\\CMS\\Model\\SiteTree.METATITLEHELP` fixes this.

**Comment by kinglozzer** [comment]
Ah yes, that’s a [known limitation](https://github.com/silverstripe/silverstripe-framework/issues/7647) of `i18nTextCollector`. Do you want to submit a PR (against `master`) to change them to `SilverStripe\\CMS\\Model\\SiteTree`? :)

**Comment by brandoniffert** [comment]
[Submitted](https://github.com/kinglozzer/silverstripe-metatitle/pull/13)

Thank you for the quick response

**Comment by kinglozzer** [comment]
Thanks! I’ve merged and tagged as `2.0.4`
